Birdee/Parser.cpp

```diff
@@ -1238,13 +1251,16 @@ BD_CORE_API bool ParseClassBody(ClassAST* ret)
 	};
 	AccessModifier access;
 	while (tokenizer.CurTok == tok_newline) tokenizer.GetNextToken();
-	std::vector<string> anno;
```

Comment:
we should allow the same annotation to be called for multiple times. e.g. If a script is called with side-effects, the calls cannot be merged to one.